src/style/cross_faded.js

```diff
@@ -3,6 +3,9 @@
 export type CrossFaded<T> = {
     from: T,
     to: T,
+    min: T,
+    mid: T,
+    max: T,
```

Comment:
when I implement the remaining `*-pattern` properties, I will be able to remove the `from` and `to` components from this type in favor of the `StyleLayer` getter for cross fade parameters.

Reply:
It looks like it should be possible now for `CrossFaded<T>` to be pared down to `{min: T, mid: T, max: T}`.  Maybe adding `from: 'min' | 'max'` to `CrossfadeParameters` so that in places where we're doing `pattern.from`, we can do `pattern[crossfade.from]` instead?